lib/schemas/Sass.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "kendo-cli://schemas/Sass.json",
+    "title": "Sass schemas",
+
+    "definitions": {
```

Comment:
Is this schema file expected to grow in the future with more descriptors? We link in the swatches file but we could also define the variable there.

Reply:
yes, we could indeed define the valid variables, as well as the variable value format i.e. `$*-bg` must accept color, `$*-gradient` must accept gradient, `$enable-*` must be boolean etc.